exp/addons/api/v1alpha3/clusterresourcesetbinding_types_test.go

```diff
@@ -147,7 +147,7 @@ func TestSetResourceBinding(t *testing.T) {
 			tt.resourceSetBinding.SetBinding(tt.resourceBinding)
 			exist := false
 			for _, b := range tt.resourceSetBinding.Resources {
-				if reflect.DeepEqual(b.ResourceRef, tt.resourceBinding.ResourceRef) {
+				if diff := cmp.Diff(b.ResourceRef, tt.resourceBinding.ResourceRef); diff == "" {
```

Comment:
```suggestion
				if cmp.Equal(b.ResourceRef, tt.resourceBinding.ResourceRef) {
```

How about this?

(same in the other 2 clusterresourcesetbingins_types_test.go files)